src/main/java/net/minecraftforge/event/TickEvent.java

```diff
@@ -51,6 +54,14 @@ public boolean haveTime()
         {
             return this.haveTime.getAsBoolean();
         }
+        
+        /**
+         * @return Returns the {@link MinecraftServer} that is currently ticking.
+         */
+        public MinecraftServer getServer()
+		{
+			return server;
+		}
```

Comment:
Formatting: indentation is 4 spaces, not tabs.
```suggestion
        {
            return server;
        }
```